OPAL/tac/src/main/resources/reference.conf

```diff
@@ -1558,5 +1558,43 @@ org.opalj {
         "org.opalj.tac.fpcf.analyses.fieldaccess.reflection.ReflectionRelatedFieldAccessesAnalysisScheduler",
       ]
     }
+
+    android{
+        AndroidEntryPointsFinder{
+            # These methods are considered entry points for an Android app.
+            "android/app/Activity" = [
+                {name = "onCreate", descriptor = "(Landroid/os/Bundle;)V"}
+                {name = "onRestart", descriptor = "()V"},
+                {name = "onStart", descriptor = "()V"},
+                {name ="onResume", descriptor = "()V"},
```

Comment:
Formatting. (Not the only place in this file where either this or the indentation is wrong)